cvat/apps/engine/tests/test_rest_api.py

```diff
@@ -40,7 +40,7 @@
 from cvat.apps.engine.tests.utils import get_paginated_collection
 from utils.dataset_manifest import ImageManifestManager, VideoManifestManager
```

Comment:
Could you add at least 1 test for this functionality? It can be done in this file, using the [`self.settings()`](https://docs.djangoproject.com/en/4.2/topics/testing/tools/#django.test.SimpleTestCase.settings) context manager.